backend/layers/persistence/orm.py

```diff
@@ -68,5 +67,5 @@ class DatasetArtifactTable:
     __tablename__ = "DatasetArtifact"
 
     id = Column(UUID(as_uuid=True), primary_key=True)
-    type = Column(Enum(DatasetArtifactType))
+    type = Column(String)
```

Comment:
we can do this in a follow up PR, but it might make sense to use string literals that way you still have some type enforcement but passing strings makes it more flexible for db / API layer.

https://mypy.readthedocs.io/en/stable/literal_types.html